### PR TITLE
AMPs for emails and phones

### PR DESCRIPTION
**Pull request by enriquepablo**
AMPs for emails and phones

### CODE REVIEW

File: eduid_proofing_amp/__init__.py [hunk 1]
```diff
@@ -64,7 +65,42 @@ def __init__(self, db_uri):
...
+    Private data for this AM plugin.
+    """
+
+    def __init__(self, db_uri):
+        self.userdb = EmailProofingUserDB(db_uri)
+        self.WHITELIST_SET_ATTRS = (
+            # TODO: Arrays must use put or pop, not set, but need more deep refacts
+            'mail',  # Old format
+            'mailAliases',  # Old format
+            'mail_addresses',
```
Comment (johanlundberg) [inline eduid_proofing_amp/__init__.py]: The attribute in the data dict is still called mailAliases.

File: eduid_proofing_amp/__init__.py [hunk 2]
```diff
@@ -64,7 +65,42 @@ def __init__(self, db_uri):
...
+    """
+    Private data for this AM plugin.
+    """
+
+    def __init__(self, db_uri):
+        self.userdb = PhoneProofingUserDB(db_uri)
+        self.WHITELIST_SET_ATTRS = (
+            # TODO: Arrays must use put or pop, not set, but need more deep refacts
+            'mobile',  # Old format
+            'phone_numbers',
```
Comment (johanlundberg) [inline eduid_proofing_amp/__init__.py]: As above, the attribute in the data dict is called mobile.

File: eduid_proofing_amp/__init__.py [hunk 3]
```diff
@@ -64,7 +65,42 @@ def __init__(self, db_uri):
...
+class EmailProofingAMPContext(object):
+    """
+    Private data for this AM plugin.
+    """
+
+    def __init__(self, db_uri):
+        self.userdb = EmailProofingUserDB(db_uri)
+        self.WHITELIST_SET_ATTRS = (
+            # TODO: Arrays must use put or pop, not set, but need more deep refacts
+            'mail',  # Old format
```
Comment (johanlundberg) [inline eduid_proofing_amp/__init__.py]: mail is no longer an attribute for the new users.

**Review by johanlundberg — CHANGES_REQUESTED** [review]

File: eduid_proofing_amp/__init__.py (hunk 3 above)
Comment (enriquepablo) [reply eduid_proofing_amp/__init__.py]: Yes I mistook the attributes

**Review by johanlundberg — APPROVED** [review]

**Pull request merged**